src/elements/teaser-product/teaser-product-static/teaser-product-static.visual.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { html, nothing, type TemplateResult } from 'lit';
+
+import { describeViewports, describeEach, visualDiffDefault } from '../../core/testing/private.js';
+
+import './teaser-product-static.js';
+import '../../button/button.js';
+import '../../image.js';
+import '../../title.js';
+
+const imageUrl = import.meta.resolve('../../core/testing/assets/placeholder-image.png');
+
+const content = (): TemplateResult => html`
+  <div>
+    <sbb-title level="3" style="margin-block-start: 0;">Benefit from up to 70% discount</sbb-title>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent pretium felis sit amet felis
+    viverra lacinia.
+    <div style="margin-block-start: var(--sbb-spacing-responsive-xxs);">
+      <sbb-button>Label</sbb-button>
```

Comment:
I'd use an action group with two buttons here (maybe static and not static visual regression tests are mixed somehow? )